components/world_news_api/actions/search-news/search-news.mjs

```diff
@@ -0,0 +1,124 @@
+import app from "../../world_news_api.app.mjs";
+
+export default {
+  name: "Search News",
+  description: "Search and filter news. [See the docs here](https://worldnewsapi.com/docs/#Search-News). **Calling this endpoint requires 1 point**",
+  key: "world_news_api-search-news",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    text: {
+      propDefinition: [
+        app,
+        "text",
+      ],
+    },
+    sourceCountries: {
+      propDefinition: [
+        app,
+        "sourceCountries",
+      ],
+    },
+    language: {
+      propDefinition: [
+        app,
+        "language",
+      ],
+    },
+    minSentiment: {
+      propDefinition: [
+        app,
+        "minSentiment",
+      ],
+    },
+    maxSentiment: {
+      propDefinition: [
+        app,
+        "maxSentiment",
+      ],
+    },
+    earliestPublishedDate: {
+      propDefinition: [
+        app,
+        "earliestPublishedDate",
+      ],
+    },
+    latestPublishedDate: {
+      propDefinition: [
+        app,
+        "latestPublishedDate",
+      ],
+    },
+    newsSources: {
+      propDefinition: [
+        app,
+        "newsSources",
+      ],
+    },
+    authors: {
+      propDefinition: [
+        app,
+        "authors",
+      ],
+    },
+    entities: {
+      propDefinition: [
+        app,
+        "entities",
+      ],
+    },
+    locationFilter: {
+      propDefinition: [
+        app,
+        "locationFilter",
+      ],
+    },
+    sort: {
+      propDefinition: [
+        app,
+        "sort",
+      ],
+    },
+    sortDirection: {
+      propDefinition: [
+        app,
+        "sortDirection",
+      ],
+    },
+    offset: {
+      propDefinition: [
+        app,
+        "offset",
+      ],
+    },
+    number: {
+      propDefinition: [
+        app,
+        "number",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const params = {
+      "text": this.text,
+      "source-countries": this.sourceCountries?.join(","), // comma-separated list
```

Comment:
```suggestion
      "source-countries": this.sourceCountries?.join(","),
```
I don't think this comment is necessary. It is pretty clear what the code is doing here